nls/platform.properties

```diff
@@ -171,6 +171,7 @@ creation.missing.clusterSelector.label.field= Label must be set
 creation.missing.clusterSelector.value = Cluster matching labels section missing label value.
 creation.missing.clusterSelector.value.field= Value must be set
 creation.missing.input = This value is required: '{0}'
+creation.input.must.not.change = This value must be: '{0}'
```

Comment:
@jeswanke Since the fix still allows you to edit the value but prevents you from saving the change due to validation, I agree we need to include the correct string here. I think we should explain why though and say that it cannot be modified or that it's read only? Perhaps @swopebe can help with the word-smithing.

For context:
![image](https://user-images.githubusercontent.com/42188127/122430853-f094e800-cf61-11eb-836d-e5064f713812.png)

Reply:
sure, let's @swopebe decide

Reply:
Slack discussion bt the three:

What about. “This value cannot be modified. Restore the original value in the YAML view to continue: ‘name’”